evap/staff/templates/staff_semester_preparation_reminder.html

```diff
@@ -10,6 +10,16 @@
 {% block content %}
     {{ block.super }}
 
+    {% csrf_token %}
+
+    {% if responsible_list %}
+        <div class="card card-submit-area text-center mb-3">
+            <div class="card-body">
+                <button id="remindAllButton" onClick="remindAllModalShow()" class="btn btn-primary">{% trans 'Remind All' %}</button>
+            </div>
+        </div>
```

Comment:
Just about the looks: Up to this point, I think we always had buttons stack up from the left without a card (compare e.g. the user list or the course type list). This looks a bit different. @janno42 do we want it this way or should it be similar to the other buttons we have?

Reply:
yes, please put the button left-aligned on top of the page (like, e.g., on `staff/user/`).